frontend/src/app/services/i18n.service.ts

```diff
@@ -0,0 +1,20 @@
+import {Injectable} from '@angular/core';
+import {marker as _} from '@biesbjerg/ngx-translate-extract-marker';
+import {TranslateService} from '@ngx-translate/core';
+import {FormattingService} from './formatting.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class I18nService extends FormattingService {
+
+  protected milKey = _(' mil.');
+  protected bilKey = _(' mld.');
+
+  constructor(
+    translateService: TranslateService,
+  ) {
+    super(translateService);
+  }
```

Comment:
Is this necessary? The super-constructor should be called by default.

Reply:
Yes. There is TranslateService injected, so we cannot use implicit constructor.